src/app/admin/volunteers/volunteers-list/volunteers-list.component.ts

```diff
@@ -163,10 +169,10 @@ export class VolunteersListComponent implements OnInit {
   }
 
   openNewVolunteerDialog() {
-    const dialogRef = this.matDialog.open(VolunteersDetailsComponent, {
-      data: {},
-      maxWidth: '100%',
-      maxHeight: '90vh',
+    const dialogRef = this.matDialog.open(NewVolunteerRegisterFormComponent, {
+      width: '555px',
+      height: '881px',
```

Comment:
Please no hardcoded dialog sizes, it should be responsive based on content

Reply:
I will discuss with Andrei Valcov to comply with Figma

Reply:
As requested changed to: 
maxWidth: '50%',
height: '90vh',